package.json

```diff
@@ -2,20 +2,22 @@
   "name": "jest-clean-console-reporter",
   "version": "0.2.0",
   "description": "Jest reporter for grouping and filtering intrusive console warnings",
-  "main": "lib/index.js",
+  "main": "lib/index.ts",
   "repository": "https://github.com/jevakallio/jest-clean-console-reporter",
   "author": "Jani Eväkallio",
   "license": "MIT",
   "scripts": {
     "prepare": "npm-install-peers"
   },
   "dependencies": {
-    "chalk": ">=3.0.0"
+    "chalk": "^3.0.0"
```

Comment:
Are these changes necessary? 

The `>=` are intentional, as `^3.0.0` does not satisfy `chalk` 4.x which is what Jest 26 uses, and similarly `^25.1.0` for Jest does not satisfy the newer version range.

Reply:
No these are not. I was messing around because when I would yarn install it would install.everything except chalk and then would complain chalk is not installed. Does that happen for you?